pkg/cmd/roachtest/monitor.go

```diff
@@ -46,7 +102,13 @@ type monitorImpl struct {
 	monitorGroup *errgroup.Group // monitor goroutine
 	monitorOnce  sync.Once       // guarantees monitor goroutine is only started once
 
-	expDeaths int32 // atomically
+	// expExactProcessDeath if true indicates that the monitor should expect exact process
+	// deaths rather than the number of process deaths. The former is a stronger assertion
+	// used in the new global roachtest monitor, while the latter should be removed
+	// when the cluster monitor is deprecated.
+	expExactProcessDeath bool
+	expDeaths            int32 // atomically
```

Comment:
Should we add `// Deprecated:` since we expect to phase it out in the near future?